vue/src/components/ImagesDownloadDialog.vue

```diff
@@ -19,8 +19,10 @@ const emit = defineEmits<{
     (e: "cancel"): void;
 }>();
 
-const baseList = ref(['S2', 'WV', 'L8', 'PL'])
-const selectedSource: Ref<Constellation[]> = ref(['WV']);
+const constellationChoices = ref(['S2', 'WV', 'L8', 'PL'])
+const selectedConstellation: Ref<Constellation[]> = ref(['WV']);
+const imageSourceChoices = computed<string[] | null>(() => selectedConstellation.value.includes('WV') ? ['cog', 'nitf'] : null);
+const selectedImageSource = ref<'cog' | 'nitf' | undefined>(undefined);
```

Comment:
Without a default value for WV you get an error.  This is related to your `@validator` not working for worldViewSource.

Reply:
Added default https://github.com/ResonantGeoData/RD-WATCH/pull/472/commits/ed700e6f07ad83a2f712ab489d118403e98dbfce